### DISCUSSION TITLE
Github page loading slow

### DISCUSSION THREAD

**Discussion by ghost** [community/community#38484] [Other Feature Feedback, Questions, & Ideas]
![image](https://user-images.githubusercontent.com/74364667/200343660-6e39641d-6b1d-489b-90cb-84921beb6e75.png)

![image](https://user-images.githubusercontent.com/74364667/200343815-8827cf38-05d0-4b13-9341-dc8514479483.png)

Suddenly my github page slowed down significantly.
No cause found.
I mainly use Chrome for development, but I deleted cookies and other data, rebooted, and restarted Chrome, but only the github page got a 400 error and the page loading speed was slow.

Does anyone have the same problem as me?

You can see the same error when using Edge browser.

The picture below is Github page loading slow #38484 
Problem opening developer tools from the page.

![image](https://user-images.githubusercontent.com/74364667/200353812-d5ac014b-c2c9-4c48-ac21-6be80cb3628a.png)

When I reload the page, I get the following error:
![image](https://user-images.githubusercontent.com/74364667/200351769-5740c2f3-f590-4080-ab5c-10d61f7190e2.png)

![image](https://user-images.githubusercontent.com/74364667/200351905-31030ca4-17b3-40d2-8e6f-6ee6c5242954.png)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Seung-hwan285 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by david-engelmann** [comment]
[I found a related stackoverflow post](https://stackoverflow.com/questions/31713987/github-api-throws-error-when-i-dont-make-any-github-api-calls). Haven't found a resolution but I'm still investigating

**Comment by yoannchaudet** [comment]
I assume you are talking about GitHub.com and not GitHub Pages (that's a product at GitHub for hosting static sites), right?

**Reply by ghost** [reply]
@yoannchaudet Yes, that's right. The problem is with the page on github.com, not the product for hosting.

I'd like to know if I'm the only one having this problem.

**Reply by sayers67** [reply]
I have this problem too, and have for awhile now

**Reply by ghost** [reply]
The problem is now gone. thanks for your answer.

**Comment by futuresea-dev** [comment]
Hello, 

Plz. reference this link

https://stackoverflow.com/questions/69790834/image-loading-slow-on-github-pages-website

**Reply by ghost** [reply]
thanks bro!
